components/Chrome.tsx

```diff
@@ -43,8 +38,19 @@ const Chrome: React.FC<Props> = ({ page, smallFooter, children }) => {
     );
   }
 
+  function toggleLanguage(): void {
+    if (language === 'en') {
+      setLang('es');
+      setCookie('languageOverride', 'es');
```

Comment:
what's the expiration on this cookie? whatever the lib gives by default? it's not just for the session, right? i'm assuming we'd like to keep the cookie around a while, maybe as long as possible...